src/hotspot/cpu/riscv/sharedRuntime_riscv.cpp

```diff
@@ -1406,13 +1406,14 @@ nmethod* SharedRuntime::generate_native_wrapper(MacroAssembler* masm,
 
   const Register ic_reg = t1;
   const Register receiver = j_rarg0;
+  const Register tmp_reg = t2;
```

Comment:
No need to introduce 'tmp_reg' here. I think you can use 't2' directly like the change made in file: src/hotspot/cpu/riscv/riscv.ad